0.data_analysis_and_processing/mic_analysis/nbconverted/well_sirna_mic_analysis.py

```diff
@@ -2,17 +2,16 @@
 # coding: utf-8
 
 # # Evaluate Well Stain Relationship Strength
-# This analysis compares Maximal Information Coefficient (MIC) scores between the same DAPI and nuclear speckle stain (A647 or GOLD) aggregated well features per well.
+# This analysis compares Maximal Information Coefficient (MIC) scores between DAPI and nuclear speckle stain (A647 and GOLD) aggregated well features per well.
 # Distributions of these MIC scores are visualized, between zero and one, where one indicates a perfect relationship and zero indicates no relationship.
-# From this analysis we can understand the relationships between the DAPI and nuclear speckle stains per cell population (Treatment, Well Position, Plate, etc...)
+# From this analysis we can understand the relationships between these stains per cell population (Treatment, Well Position, Plate, etc...)
```

Comment:
I love these, this sums up this script very well!

Reply:
Thanks!